tools/api-markdown-documenter/src/documentation-domain/TextFormatting.ts

```diff
@@ -14,19 +14,19 @@ export interface TextFormatting {
 	 *
 	 * @defaultValue Inherit
 	 */
-	readonly italic?: boolean;
+	readonly italic?: true;
```

Comment:
As originally typed, this gave the impression that you could explicitly _disable_ these options at some lower scope in the tree, which is not the case. The options are `true` or `inherit`.

Reply:
Not sure what `inherit` means in this context... aren't the types for these unconditionally `true` no matter what?

Reply:
Since they're optional, the types end up being `true | undefined`.

Inherit in this context means "inherit from some ancestral span that set formatting". I'll elaborate on this in the docs.

On further consideration, though, it occurs to me that it really _is_ valid to explicitly disable formatting at some lower scope. The system doesn't currently leverage this degree of freedom, but it is valid. So I think I'll revert the type changes here. But I'll update the docs to make the semantics of "inherit" more clear.